app/Http/Controllers/OrderTrackingController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Order;
+use Illuminate\Http\Request;
+
+class OrderTrackingController extends Controller
+{
+    public function index(){
+        $orders = Order::with('orderLine', 'orderStatus')->where('user_id', auth()->id())->get();
+
+        //Calculate most expensive product for each order and add to Associative array
```

Comment:
Het bepalen hiervan kan je beter in de Order model doen, zo heb je een betere SoC. 
Uiteindelijk kan je dan gewoon dit doen:
`$mostExpensiveProductsByOrder = $orders->mapWithKeys(function ($order) {
            return [$order->id => $order->getMostExpensiveProduct()];
});`